src/pages/Home.js

```diff
@@ -4,8 +4,11 @@ import Cards from "./home/Cards"
 export const Home = () => {
     return (
         <>
-            <HeroSection/>
-            <Cards/>
+            <HeroSection />
+            <section id="games-section" style={{ padding: '100px 0' }}>
+                <h2>Games Section</h2>
```

Comment:
The cards below show both games and activities so naming this just games would be incorrect
Also the text is kinda invisible cuz of the background
![image](https://github.com/user-attachments/assets/69cebac8-3c25-4b06-88eb-99bd385624e8)